config/initializers/0_clowder-config.rb

```diff
@@ -19,13 +19,32 @@
   redis_url = "#{config.dig('inMemoryDb', 'hostname')}:#{config.dig('inMemoryDb', 'port')}"
   redis_password = config.dig('inMemoryDb', 'password')
 
+  # Kafka
+  first_kafka_server_config = config.kafka.brokers[0]
+  kafka_security_protocol = first_kafka_server_config.dig('authtype')
+
+  kafka_server_config = {
+    brokers: config.dig('kafka', 'brokers')&.map { |b| "#{b&.dig('hostname')}:#{b&.dig('port')}" }.join(',')
+  }
+
+  if kafka_security_protocol
+    if kafka_security_protocol == 'sasl'
+      kafka_server_config['security_protocol'] = 'sasl_ssl'
```

Comment:
Nit: `kafka_server_config` started with Symbols as key. This is not a problem, as the Settings load would map it correctly.

Reply:
Sorry, as a Ruby noob I didn't even tell the difference at first, thanks ! I updated all the string keys to symbols